src/index.js

```diff
@@ -41,7 +41,7 @@ import PaymentSuccess from './components/pages/ParentHome/PaymentSuccess';
 import Cart from './components/pages/ParentHome/Cart';
 import ParentFamilyHome from './components/pages/ParentFamily/ParentFamilyHome';
 import NavBar from './components/common/NavBar';
-import Newfeed from './components/pages/n';
+// import Newfeed from './components/pages/n';
```

Comment:
Did you mean to comment this out?

Reply:
I don't see it being implemented inside `src/index.js` file so I thought whoever is in charge of importing this component did not finish writing down the complete relative path. 

I also couldn't find the component itself inside the `src/components/pages` folder so I decided to just comment it out to prevent the code from breaking.